tests/django/conftest.py

```diff
@@ -1,179 +1,46 @@
-        enable_db=False,
-    )
-
-
-@pytest.fixture()
-def correct_auth_profile(rf, brp_r_profile):
-    # Correct: both scope and mandatory query parameters in the request.
-    correct_request = rf.get(
-        "/",
-        data={
-            "postcode": "1234AB",
-            "lastname": "Foobar",
-        },
-    )
-    correct_request.is_authorized_for = lambda *scopes: "BRP/R" in set(scopes)
-    return RequestProfile(correct_request)
-
-
-@pytest.fixture()
-def incorrect_auth_profile(rf, brp_r_profile):
-    # Incorrect: no queries are given, so no access is given.
-    incorrect_request = rf.get(
-        "/",
-        data={
-            "postcode": "1234AB",
-        },
-    )
-    incorrect_request.is_authorized_for = lambda *scopes: "BRP/R" in set(scopes)
-    auth_profile = RequestProfile(incorrect_request)
-    return auth_profile
-
-
-@pytest.fixture()
-def afval_dataset(afval_schema: DatasetSchema) -> Dataset:
-    """Create Afvalwegingen dataset."""
-    return Dataset.create_for_schema(afval_schema)
-
-
-@pytest.fixture()
-def brk_dataset(brk_schema: DatasetSchema) -> Dataset:
-    """Create full BRK virtual Databset."""
-    dataset = Dataset.create_for_schema(brk_schema)
-    dataset.enable_db = False
-    dataset.save()
-    return dataset
-
-
-@pytest.fixture()
-def gebieden_dataset(gebieden_schema: DatasetSchema) -> Dataset:
-    """Create gebieden dataset. DO NOT USE TOGETHER WITH ggwgebieden_dataset."""
-    return Dataset.create_for_schema(gebieden_schema)
-
-
-@pytest.fixture()
-def ggwgebieden_dataset(ggwgebieden_schema: DatasetSchema) -> Dataset:
-    """Create ggwgebieden dataset. DO NOT USE TOGETHER WITH gebieden_dataset."""
-    return Dataset.create_for_schema(ggwgebieden_schema)
-
-
-@pytest.fixture()
-def hr_dataset(hr_schema: DatasetSchema) -> Dataset:
-    """Create HR dataset."""
-    return Dataset.create_for_schema(hr_schema)
-
-
-@pytest.fixture()
-def meetbouten_dataset(meetbouten_schema: DatasetSchema) -> Dataset:
-    """Create Meetbouten dataset."""
-    return Dataset.create_for_schema(meetbouten_schema)
-
-
-@pytest.fixture()
-def meldingen_dataset(meldingen_schema: DatasetSchema) -> Dataset:
-    """Create Meldingen dataset."""
-    return Dataset.create_for_schema(meldingen_schema)
-
-
-@pytest.fixture()
-def parkeervakken_dataset(parkeervakken_schema: DatasetSchema) -> Dataset:
-    """Create Parkeervakken dataset."""
-    return Dataset.create_for_schema(parkeervakken_schema)
-
-
-@pytest.fixture()
-def verblijfsobjecten_dataset(verblijfsobjecten_schema: DatasetSchema) -> Dataset:
-    """Create Verblijfsobjecten dataset."""
-    return Dataset.create_for_schema(verblijfsobjecten_schema)
-
-
-@pytest.fixture()
-def woningbouwplannen_dataset(woningbouwplannen_schema: DatasetSchema) -> Dataset:
-    """Create Woning Bouw Plannen dataset."""
-    return Dataset.create_for_schema(woningbouwplannen_schema)
+    config.pluginmanager.register(tests.django.fixtures)
```

Comment:
I struggled to get this to work after you pointed out that you could not get individual Django tests to run anymore after my restructuring changes to the `schematools` package.

So this is what it takes?
* explicit call to   `_setup_django()`
* and registering a new module containing the fixtures as a plugin

Very nice and straightforward! :+1: I learned something new!

Reply:
Pretty much yes.

`_setup_django()` is actually called by `pytest-django` later on, but it would be too late for us.
And trick with module registration is needed because otherwise fixtures won't be recognized.w